app/scenes/Schedule/index.js

```diff
@@ -15,7 +15,7 @@ import moment from 'moment';
 
 import type { ScheduleTalk } from '../../types';
 
-import Splash from 'react-native-smart-splash-screen';
+// import Splash from 'react-native-smart-splash-screen';
```

Comment:
@brentvatne, thanks for that! I'll pull in some of those changes, but for example I don't see a need for an `Animated.ListView` as the list view never moves. Let me know if I'm missing something.